src/lib/mlang.js

```diff
@@ -339,6 +339,46 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.mlang.predicate',
+  name: 'InIC',
+  extends: 'foam.mlang.predicate.Binary',
+
+  properties: [
+    {
+      name: 'arg2',
```

Comment:
We do; see how valueSet_ is constructed below. We can't pre-uppercase if `arg2` is not an array; it might be projected from the input object.